### PR TITLE
fix: Instance of with correct duration names

### LINKED ISSUE #155: instance of doesn't work for durations
**Describe the bug**
I can not check the type of a duration value using `instance of`.

**To Reproduce**
Expression `null instance of years and months duration` produces a failure but should return `false`

**Expected behavior**
```
null instance of years and months duration -> false
null instance of days and time duration -> false

duration("P3M") instance of years and months duration -> true
duration("PT4H") instance of days and time duration -> true
```

**Environment**
* FEEL engine version: 1.11.2

### PR DESCRIPTION
**Pull request by sccalabr**
## Description
*  instance of doesn't work for durations 

Expected behavior

null instance of years and months duration -> false
null instance of days and time duration -> false

duration("P3M") instance of years and months duration -> true
duration("PT4H") instance of days and time duration -> true

## Related issues

closes #155

### CODE REVIEW

**Comment by saig0** [comment]
@sccalabr thank you for your contribution. :tada:  I'll have a look at your PR in the next few days. :eyes:

**Comment by sccalabr** [comment]
Does this look good?

File: docs/docs/reference/language-guide/feel-boolean-expressions.md [hunk 1]
```diff
@@ -230,6 +230,12 @@ Use the type `Any` to check if the value is not `null`.
 
 null instance of Any
 // false
+
+"duration("P3M") instance of years and months duration"
+// true
+
+"duration("PT4H") instance of days and time duration"
+// true
```
Comment (saig0) [inline docs/docs/reference/language-guide/feel-boolean-expressions.md:238]: :x: The expression is wrapped into double quotes.

```suggestion
duration("P3M") instance of years and months duration
// true

duration("PT4H") instance of days and time duration
// true
```

Comment (saig0) [reply docs/docs/reference/language-guide/feel-boolean-expressions.md:238]: :x: We should update the type names in the description above. 

We can replace the old names for the duration types. A user should use the official type names.

**Review by saig0 — APPROVED** [review]
@sccalabr your changes look good. :tada:

I have a minor comment about the documentation. I'll adjust the docs myself and merge the PR. :rocket: 

---

This pull request counts toward the Hacktoberfest challenge. If you contributed four pull requests then you can complete your Camunda Hacktoberfest challenge [here](https://camunda.com/hacktoberfest2022/).

**Pull request merged**